constants/creator_groups_by_country.json

```diff
@@ -1,477 +1,428 @@
-        "Johns Hopkins University",
-        "Kno.e.sis",
-        "Leetcode",
-        "Machine Intelligence Research Institute",
-        "Massachusetts Institute of Technology",
-        "Megagon Labs",
-        "Meta",
-        "MetaMind - A Salesforce Company",
-        "Michigan State University",
-        "Microsoft",
-        "Microsoft Research",
-        "Microsoft Semantic Machines",
-        "Monash University",
-        "New York University",
-        "NomicAI",
-        "Northeastern University",
-        "Northwestern University",
-        "OpenAI",
-        "Paul G. Allen School of Computer Science & Engineering",
-        "Penn State University",
-        "Pennsylvania State University",
-        "Princeton University",
-        "Reed College",
-        "Rensselaer Polytechnic Institute",
-        "Rollins College",
-        "Rutgers University",
-        "S.A.I.C",
-        "Salesforce Research",
-        "ShareGPT",
-        "Snap Inc.",
-        "Snips",
-        "Stanford University",
-        "Stony Brook University",
-        "Surge AI",
-        "Swinburne University of Technology",
-        "TaskRabbit",
-        "Texas A&M University",
-        "The Institute for Human & Machine Cognition",
-        "The Nueva School",
-        "The Ohio State University",
-        "The Pennsylvania State University",
-        "The University of Texas at Austin",
-        "Toyota Technological Institute at Chicago",
-        "TransPerfect Translations",
-        "Tulane University",
-        "Twitter",
-        "U.S. Army Research Laboratory",
-        "UC Berkeley",
-        "UC Irvine",
-        "UC San Diego",
-        "UC San Francisco",
-        "UC Santa Barbara",
-        "UC Santa Cruz",
-        "UCLA",
-        "UIUC",
-        "UMass Amherst",
-        "UNC Chapel Hill",
-        "US Naval Academy",
-        "United States Naval Academy",
-        "University of Arizona",
-        "University of Central Florida",
-        "University of Chicago",
-        "University of Colorado Boulder",
-        "University of Houston",
-        "University of Illinois",
-        "University of Illinois Urbana-Champaign",
-        "University of Maryland",
-        "University of Massachusetts Amherst",
-        "University of Massachusetts Lowell",
-        "University of Michigan",
-        "University of Oregon",
-        "University of Pennsylvania",
-        "University of Pittsburgh",
-        "University of Rochester",
-        "University of Southern California",
-        "University of Texas at Dallas",
-        "University of Utah",
-        "University of Virginia",
-        "University of Washington",
-        "Virginia Tech",
-        "Wright State University",
-        "Yahoo! Inc.",
-        "Yale University"
-    ],
-    "Vietnam": [
-        "Vietnam Institute of Information Technology"
-    ]
-}
\ No newline at end of file
+  "African Continent": ["Alamoja Yoruba & Masakhane NLP", "Masakhane NLP"],
```

Comment:
Could we re-format as it was before? It's hard to see if there were any changes. Thanks!